### PR TITLE
enable Services Test

### PR DESCRIPTION
**Pull request by bensonlatibm**
Modified ServicesTest to check for an executable java.
Will skip test if java is not executable by openliberty.

### CODE REVIEW

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
manual build with passing results:
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_t3zhoGM2EeqDwI5U1uUYvg&tab=com.ibm.team.build.web.ui.internal.editors.result.test.TestProvider

**Comment by bensonlatibm** [comment]
Ran manual build with build.os.native.packages set to true.
Ran regular pb without build.os.native.packages set to true.

File: dev/com.ibm.ws.install.packaging_fat/fat/src/com/ibm/ws/install/packaging/fat/InstallUtilityToolTest.java [hunk 1]
```diff
@@ -32,6 +32,7 @@
     public static boolean connectedToRepo = true;
```
Comment (alvinso) [inline dev/com.ibm.ws.install.packaging_fat/fat/src/com/ibm/ws/install/packaging/fat/InstallUtilityToolTest.java]: We should rename InstallUtilityToolTest.java

Comment (bensonlatibm) [reply dev/com.ibm.ws.install.packaging_fat/fat/src/com/ibm/ws/install/packaging/fat/InstallUtilityToolTest.java]: will use 11284 to rename this

**Comment by bensonlatibm** [comment]
#build

**Comment by bensonlatibm** [comment]
#run-libby-bot

**Pull request merged**